packages/vue-labs/src/components/FTable/table-column.ts

```diff
@@ -37,6 +39,7 @@ export interface TableColumnRowHeader<T, K extends keyof T> {
 export interface NormalizedTableColumnRowHeader<T, K> {
     readonly type: "rowheader";
     readonly header: string;
+    readonly description?: string;
```

Comment:
```suggestion
    readonly description: string | null;
```

Gäller samtliga. Motiveringen är att då kan vi förlita oss på att `description` egenskapen alltid kommer finnas (dvs den finns där men den kan vara `null`).